integration/network/bridge/bridge_linux_test.go

```diff
@@ -193,3 +196,124 @@ func TestIPRangeAt64BitLimit(t *testing.T) {
 		})
 	}
 }
+
+// TestFilterForwardPolicy tests that, if the daemon enables IP forwarding on the
+// host, it also sets the iptables filter-FORWARD policy to DROP (unless it's
+// told not to).
+func TestFilterForwardPolicy(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "no bridge driver or iptables on Windows")
```

Comment:
```suggestion
```

That's not needed, we're in `bridge_linux_test.go`.

Reply:
Good point!